### ISSUE TITLE
setting up a newly detected folder for auto upload does not upload already existing pictures

### ISSUE THREAD

**Issue by lemmy04** [nextcloud/android#4625]
### Actual behaviour
the nextcloud client detects a new folder with pictures... image editing apps or alternate camera apps sometimes use a separate folder...
so I get the auto upload popup, and i mark that new folder for automatic uploads
once this is done, *new* pictures will be automatically uploaded - but the existing image(s) that initially triggered that "mark as auto upload folder" popup are NOT uploaded.

this, in combination with #4492, basically means that every time i delete the pictures off my phone I will end up with the next picture *not* being synced into my cloud.

### Expected behaviour
I expect existing content in a new autoupload folder to be uploaded... automatically.
 
### Steps to reproduce
1. install some app that creates image files in its own data folder
2. use it
3. configure the new data folder for auto upload
4. observe existing files in it NOT being synced into the clou
5. observe any further photos being synced

### Environment data
Android version: Any (happens to me on 4.4.1, 7, 8 and 9, and some LineageOS versions)

Device model: 
Several: Samsung Tab E, Samsung Tab S2, Samsung Tab S5e, Samsung Galaxy S5, S6 edge, S8+

Stock or customized system:
see above. Some devices are rooted, some are not.

Nextcloud app version:
Latest

Nextcloud server version:
Latest

**Comment by tobiasKaminsky** [comment]
This is a duplicate of 
#4430 
#3875
